infrastructure-petstore-rest-client/src/main/java/today/stepbeyond/examples/springbootexamples/infrastructure/gateways/api/model/Request.java

```diff
@@ -0,0 +1,17 @@
+package today.stepbeyond.examples.springbootexamples.infrastructure.gateways.api.model;
+
+import java.util.UUID;
+
+public class Request {
```

Comment:
What about a more precise naming? 
```suggestion
public class RegisterPetRequest {
```

Reply:
thx.